src/gui/DatabaseWidget.cpp

```diff
@@ -755,7 +755,7 @@ void DatabaseWidget::entryActivationSignalReceived(Entry* entry, EntryModel::Mod
 void DatabaseWidget::switchToEntryEdit()
 {
     Entry* entry = m_entryView->currentEntry();
-    Q_ASSERT(entry);
+    
```

Comment:
Well, t wasn't exactly a bug. It was only an empty search result and the function did what it was supposed to do: nothing. The bug was rather the assertion which crashed the program.